### PR TITLE
Allow aggregations

### PR DESCRIPTION
**Pull request by amencarini**

### CODE REVIEW

**Comment by pedrocunha** [comment]
Maybe let's add a spec that covers this ?

File: spec/lib/ht/search_client/remote_spec.rb [hunk 1]
```diff
@@ -55,6 +55,36 @@ def endpoint
...
       end
 
+      context 'with unexpected params' do
+        let(:params) { { not_a_param: 'bad value' } }
+        let(:request_url) { 'http://username:password@test.com/foo?per_page=32&page=1&currency=EUR&order=sqs_score' }
+
+        it 'should add the parameters to the query' do
+          subject.perform
+          expect(stub_request :get, request_url).to have_been_requested
+        end
```
Comment (mottalrd) [inline spec/lib/ht/search_client/remote_spec.rb]: I am struggling to understand this test

`should add the parameters to the query` => `expect(stub_request :get, request_url).to have_been_requested`

O_o could you explain it to me?

Comment (mottalrd) [inline spec/lib/ht/search_client/remote_spec.rb]: Oh, I think I got it. Maybe it's worth to say `default parameters`

Comment (amencarini) [inline spec/lib/ht/search_client/remote_spec.rb]: Those are not stated in the spec but will clarify :+1:

**Comment by mottalrd** [comment]
SInce we are here, is it worth it to update the README to explain this feature?

**Comment by amencarini** [comment]
@mottalrd good call, done

File: README.md [hunk 2]
```diff
@@ -201,6 +201,12 @@ All filters are optional. The filters you can apply are based on:
 - `price_max [integer]`
 - `currency [string] [default: 'EUR']`
 
+### Aggregations
+
+All aggregations are optional. The aggregations you can apply are based on:
+- `aggregations` as a comma-separated string (e.g.: `type_of_property,garden`)
+- `range_aggregations` as an array of comma-separated strings (e.g.: `price,-100,100-200,200-`)
```
Comment (mottalrd) [inline README.md]: so the first value in the string identify the "type?"

Comment (amencarini) [inline README.md]: It identifies the field, yes.

**Pull request merged**